app/controllers/v0/pension_claims_controller.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
 require 'pension_21p527ez/tag_sentry'
+require 'pension_21p527ez/monitor'
 
 module V0
   class PensionClaimsController < ClaimsBaseController
     service_tag 'pension-application'
 
+    MONITOR = ::Pension21p527ez::Monitor.new
```

Comment:
The scope resolution operator `::` isn't necessary. This should also be moved to a private method such as
```
def pension_monitor
  Pension21p527ez::Monitor.new
 end 
```